scripts/image_customization_lib.lua

```diff
@@ -0,0 +1,135 @@
+local M = {}
+
+local function file_exists(file)
+	local f = io.open(file)
+	if not f then
+		return false
+	end
+	f:close()
+	return true
+end
+
+local function get_customization_file_name(env)
+	return env.GLUON_SITEDIR .. '/image-customization'
```

Comment:
This is basically a lua file, right?
Shouldn't we also name it as one, so that highlighting and code linting works better?

Reply:
Would also make auto-formatting easier